src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -19,17 +20,17 @@ function getDateString(date) {
   return date ? formatDate(date, true) : 'unknown date';
 }
 
-function getProgressText(intl, startDate) {
+function getProgressText(intl, startDate, currentDate) {
```

Comment:
I switched back from houston `now` to `new Date()` in https://github.com/WildMeOrg/codex-frontend/pull/362/commits/57c6cb8bce18e5456e076fca696a2849e6dbffdd. For me the Curation time elapsed is now correct, but Detection is incorrect because of the aforementioned missing time zone designator.